src/main/java/uk/gov/hmcts/darts/log/service/impl/ArmLoggerServiceImpl.java

```diff
@@ -18,6 +21,8 @@ public class ArmLoggerServiceImpl implements ArmLoggerService {
 
     private static final String ARCHIVE_TO_ARM_FAILED = "Failed to archive object to ARM: eod_id={}";
 
+    private static final String ARM_MISSING_RESPONSE = "No response files produced by ARM within {} for EOD {}";
```

Comment:
@hemantasharma1129 this template string is parameterised in two places (duration and eod id), whereas the ticket appears to expect parameterisation only in one place i.e:
```
log.error("No response files produced by ARM within 24 hours for EOD {}", eod.getId());
```

The approach that's implemented in this PR is sensible and correct, but can you confirm that Dynatrace integration is happy with the duration being dynamic, and that they don't expect "24 hours" (or "1 day") to be a fixed constant?

Reply:
Yeah Dynatrace can handle dynamic string and 24 hours is a configurable value in application.yaml.